pkgs/desktops/gnome/core/pop-desktop-widget/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, rustPlatform, lib, pkg-config, glib, gst_all_1, gtk3, libhandy }:
+
+stdenv.mkDerivation rec {
+  pname = "pop-desktop-widget";
+  version = "unstable-2021-11-03";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "desktop-widget";
+    rev = "1652fc89aa16dcf652422c4c3dfe1095e6286b7d";
+    sha256 = "sha256-QBQBtBSj+J87yAQhYMOrJTUCee3ebJhVc4Y/7EyBDes=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
```

Comment:
Why are you not using the rustPlatform mkDerivation?

Reply:
This is because `pop-desktop-widget` uses a `Makefile` execute Cargo

Reply:
That most likely won't work with nix and we need to replicate that in the right rustPlaform function.

Reply:
We have plenty of packages that use Make or Ninja to run cargo, and they work fine.  Is there a specific problem you've noticed here?